From bf5d26ce72976670ff2f15f012af96d3f677c7c3 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Thu, 25 May 2017 16:15:04 +1000
Subject: [PATCH] Use pam security directory

Pam currently uses /lib/security for it's files, so kwallet needs
to put it there.

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bcc5057..d0e4607 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,7 +52,7 @@ target_link_libraries (${library_name}
    ${LIBGCRYPT_LIBRARIES}
 )
 
-install(TARGETS ${library_name} DESTINATION ${CMAKE_INSTALL_LIBDIR}/security)
+install(TARGETS ${library_name} DESTINATION /lib/security)
 
 configure_file(pam_kwallet_init.desktop.cmake ${CMAKE_BINARY_DIR}/pam_kwallet_init.desktop)
 
-- 
2.13.0

